### DISCUSSION TITLE
Hello, where can I see the maintenance cycle of each version of skywalking?

### DISCUSSION THREAD

**Discussion by Hello-Linux** [apache/skywalking#11109] [General]
Hello, where can I see the maintenance cycle of each version of skywalking? Or which version is the long-term support version?

**Comment by Hello-Linux** [comment]
@wallezhang  @augustowebd @dio

**Comment by wu-sheng** [comment]
Stop pinging various people. If someone knows, they may(or may not) answer.
People don't have obligations to support you.
Understand how a community works as volunteers based.
